src/util/address.rs

```diff
@@ -98,6 +100,7 @@ impl fmt::Display for Error {
             Error::InvalidSegwitV0ProgramLength(l) => write!(f, "a v0 witness program must be either of length 20 or 32 bytes: length={}", l),
             Error::UncompressedPubkey => write!(f, "an uncompressed pubkey was used where it is not allowed"),
             Error::ExcessiveScriptSize => write!(f, "Script size exceed 520 bytes"),
+            Error::UnrecognizedScript => write!(f, "Script is not a p2pk, p2sh or witness program")
```

Comment:
I believe this should be p2pkh not p2pk.
```suggestion
            Error::UnrecognizedScript => write!(f, "Script is not a p2pkh, p2sh or witness program")
```

Reply:
Ahh you're right! Thanks for pointing it out, I need more coffee 😄